### PR TITLE
AWS testing Scripts using Moto

### PR DESCRIPTION
**Pull request by shantamsultania**
this is my solution for #1101

### CODE REVIEW

File: Python/AWS testing Script/Readme.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Aws Script for testing using moto (a AWS testing framework )
+
+This python script automatically generates a set of EC2 machine's with any demo ami Id that could be used in case of testing.
+
+## Installation
+
+First of all install [python]("https://www.python.org/downloads/") on your system.
```
Comment (RohiniRG) [inline Python/AWS testing Script/Readme.md:7]: Installing python seems pretty redundant. 
Put all the packages to be installed in a `requirements.txt` file

File: Python/AWS testing Script/Readme.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+```bash
+pip install moto
+pip install boto3
+pip install Flask
+```
+
+## Results
+
+![](https://i.postimg.cc/Jzx6f02c/image1.png)
+
```
Comment (RohiniRG) [inline Python/AWS testing Script/Readme.md:17]: Try using Imgur for generating your image links and write it as `Image`

File: Python/AWS testing Script/Readme.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+![](https://i.postimg.cc/L6RVmRfy/image.png)
+
+![](https://i.postimg.cc/Zq1r2h4x/image.png)
+
```
Comment (RohiniRG) [inline Python/AWS testing Script/Readme.md:21]: Also mention how one can run the script in the README itself

File: Python/AWS testing Script/agentMain.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+
+# you can add more apis of your choice here like
+# create Volume, VPC and snapshots
+
+# to do so just add a call function in awsTester class and agentMain and you are done
+
+# run the app
+if __name__ == '__main__':
+    app.run()
```
Comment (RohiniRG) [inline Python/AWS testing Script/agentMain.py:33]: Add a newline at the end of all the files

File: Python/AWS testing Script/agentMain.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+@app.route("/ec2", methods=['POST', 'GET'])
+def ec2():
+    client = aws_testing.add_service("ec2", "us-east-1")
+    return aws_testing.test_create_ec2(client)
+
+
+# you can add more apis of your choice here like
+# create Volume, VPC and snapshots
+
```
Comment (RohiniRG) [inline Python/AWS testing Script/agentMain.py:28]: Remove redundant blank lines.

File: Python/AWS testing Script/awsHandler.py [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    client.create_snapshot(VolumeId=volume_id)
+
+
+def create_ec2_volume(client, AZ):
+    ab = client.create_volume(AZ)
+    return ab
+
+
+def create_vpc(client, cidr_block):
+    client.create_vpc(CidrBlock=cidr_block)
```
Comment (RohiniRG) [inline Python/AWS testing Script/awsHandler.py:15]: Add newline

File: Python/AWS testing Script/awsTester.py [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+    k2 = Key(bucket)
+    k2.key = key_name
+    data = k2.get_contents_as_string()
+    print('Fetched object {}/{} with content: {}'.format(bucket.name,
+                                                         key.name, data))
+
+
+if __name__ == "__main__":
+    client = add_service("ec2", "us-east-1")
+    test_create_ec2(client)
```
Comment (RohiniRG) [inline Python/AWS testing Script/awsTester.py:59]: Add newline

**Review by RohiniRG — COMMENTED** [review]
Good work! 
Add more comments to your code files and make the requested changes

**Comment by RohiniRG** [comment]
@shantamsultania any updates on this?

**Comment by shantamsultania** [comment]
> @shantamsultania any updates on this?

will be done by Sunday I am a bit busy with some work sorry for the delay @RohiniRG

**Comment by RohiniRG** [comment]
@shantamsultania are you still working on this?

**Comment by HarshCasper** [comment]
Closing the PR because of no updates.

**Pull request closed without merging**